packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1481,6 +1482,15 @@ export default class Meeting extends StatelessWebexPlugin {
      * @memberof Meeting
      */
     this.connectionStateHandler = undefined;
+
+    /**
+     * ICE Candidates errors map
+     * @instance
+     * @type {Map<[number, string], number>}
+     * @private
+     * @memberof Meeting
+     */
+    this.iceCandidateErrors = new Map();
```

Comment:
we need to clear this map on every meeting join in case same user joins, leaves and joins again the same meeting. We could do it in addMedia() or I guess we can just do it in line 6023 just before registering the listener

Reply:
Done